modular_skyrat/modules/blueshield/code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -22,3 +22,5 @@
 	new /obj/item/clothing/suit/armor/vest/blueshield(src)
 	new /obj/item/clothing/shoes/laceup(src)
 	new /obj/item/clothing/under/rank/security/blueshield(src)
+	new /obj/item/clothing/suit/armor/vest/blueshieldarmor(src)
```

Comment:
I would probably move this one to the Command vendor, at the very least. Actually, most of the Blueshield's stuff should be moved to the Command vendor.